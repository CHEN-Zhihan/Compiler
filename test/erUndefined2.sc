def main() {
    c = @a;
    return c;
}


main();