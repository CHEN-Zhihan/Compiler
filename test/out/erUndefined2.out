Undefined reference to global variable: a
